### DISCUSSION TITLE
anyOf<E, RS> what's RS?

### DISCUSSION THREAD

**Discussion by agufagit** [isar/isar#1355] [Help] [answered]
anyOf<E, RS> what's RS?  What value should I put for it?

`anyOf<String, ...>(['s', 'p'], (q, element) => q.idEqualTo(element))`

Thank you

**Comment by albrnick** [comment]
According to 1 example I saw, I don't think you need to specify a type in your anyOf call. Just use:

`anyOf(['s', 'p'], (q, element) => q.idEqualTo(element))`

Just a guess based off of one example.

**Reply by agufagit** [reply]
vscode requires type to be specified, error was "can't infer type"

**Reply by albrnick** [reply]
Man.. Whew.. I'm looking into the code, and RS doesn't seem to be used in the anyOf function.  Have you tried putting something just like void or String or something?  Maybe the error it gives would help you figure out what to put there?

**Reply by agufagit** [reply]
I put in 'dynamic'.  Yeah, I looked at the code also, just wondering what it is and what it's there for.

**Reply by simc** [reply, marked as answer]
This is indeed a bug. Thanks for reporting. Will be fixed in the next release
